ground_truth_labeling_jobs/3d_dense_point_cloud_downsampling_tutorial/ground_truth_annotation_dense_point_cloud_tutorial.ipynb

```diff
@@ -0,0 +1,744 @@
+{
```

Comment:
<p>Add description above this cell. Suggestion:</p><p> </p><p>"In the following cell, you generate a label category configuration file. This file contains the label categories workers use to label the 3D point cloud, worker instructions, and other information about your task. To learn more about all of the options you have when you create a label category configuration file, see <a href="https://docs.aws.amazon.com/sagemaker/latest/dg/sms-label-cat-config-attributes.html" target="_blank">Create a Labeling Category Configuration File with Label Category and Frame Attributes</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Updated it</p>